strawberry/cli/commands/import_schema.py

```diff
@@ -0,0 +1,64 @@
+import re
+from typing import List
+
+import click
+
+from graphql.error import GraphQLError
+from graphql.language import parse
+
+from ...utils.sdl_transpiler import render_template
+
+
+# Regex pattern to extract import statement
+GQL_IMPORT_REGEX = r"#import \"(.*\.gql|.*\.graphql)\""
+GQL_SCHEMA_TYPE = r"(?si)^Schema.*?}"
+
+
+@click.command("import-schema", short_help="Parses SDL file to python dataclasses")
+@click.argument("schema", type=str, required=True, nargs=-1)
+def import_schema(schema):
+    """ Parses SDL file to python dataclasses and writes them out """
+
+    templates = []  # Put rendered templates here
+    try:
+        ast = get_ast(schema)  # Get abstract syntax tree
+        for definition in ast.definitions:
+            # Parse and render specific ast definitions
+            templates.append(render_template(definition))
+
+        imports = "import strawberry"
+        schema_string = "\n\n".join(templates)
+        imports += "\nimport typing\n\n" if "typing" in schema_string else ""
+        schema_string = imports + schema_string
+        click.echo(schema_string)
+
+    except FileNotFoundError:
+        print(f"File or files not found on schema path: {schema}")
+
+    except GraphQLError:
+        # TODO: Test this to see if print output can be improved
+        print("A file contains syntax errors")
```

Comment:
can we move all this logic to strawberry.utils.schema_importer[1]?

[1] not sure about the name for the module, but we can deal with it later :)